securitycenter/snippets_v2/snippets_notification_configs_v2.py

```diff
@@ -0,0 +1,224 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Demos for working with notification configs."""
+
+from typing import Dict
+
+# [START securitycenter_create_notification_config_v2]
+def create_notification_config(parent_id, location_id, pubsub_topic, notification_config_id) -> Dict:
+    """
+    This method is used to create the Notification Config.
+    Args:
+        parent_id: must be in one of the following formats:
+            "organizations/{organization_id}"
+            "projects/{project_id}"
+            "folders/{folder_id}"
+        location_id: "global"
+        pubsub_topic: "projects/{your-project-id}/topics/{your-topic-ic}"
+        notification_config_id: "your-config-id"
+
+
+    Ensure this ServiceAccount has the "pubsub.topics.setIamPolicy" permission on the new topic.
+    """
+    from google.cloud import securitycenter_v2 as securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+    parent_id = parent_id+"/locations/"+location_id
+    response = client.create_notification_config(
+        request={
+            "parent": parent_id,
+            "config_id": notification_config_id,
+            "notification_config": {
+                "description": "Notification for active findings",
+                "pubsub_topic": pubsub_topic,
+                "streaming_config": {"filter": 'state = "ACTIVE"'},
+            },
+        }
+    )
+    # [END securitycenter_create_notification_config_v2]
+    return response
+
+
+# [START securitycenter_delete_notification_config_v2]
+def delete_notification_config(parent_id, location_id, notification_config_id) -> bool:
+    """
+    This method is used to delete the Notification Config.
+    Args:
+        parent_id: must be in one of the following formats:
+            "organizations/{organization_id}"
+            "projects/{project_id}"
+            "folders/{folder_id}"
+        location_id: "global"
+        notification_config_id: "your-config-id"
+    """
+    from google.cloud import securitycenter_v2 as securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+    parent_id = parent_id+"/locations/"+location_id
+    notification_config_name = (
+        f"{parent_id}/notificationConfigs/{notification_config_id}"
+    )
+
+    client.delete_notification_config(request={"name": notification_config_name})
+    print(f"Deleted notification config: {notification_config_name}")
+    # [END securitycenter_delete_notification_config_v2]
+    return True
```

Comment:
I don't see the value in returning True here. Please remove this.
In testcases, you can assert failure by trying to retrieve the notification config.

Reply:
addressed